app/stac_api/models.py

```diff
@@ -483,6 +487,31 @@ class Item(models.Model):
 
     class Meta:
         unique_together = (('collection', 'name'),)
+        indexes = [
+            models.Index(fields=['name'], name='item_name_idx'),
+            models.Index(fields=['properties_datetime'], name='item_datetime_idx'),
+            models.Index(fields=['properties_start_datetime'], name='item_start_datetime_idx'),
+            models.Index(fields=['properties_end_datetime'], name='item_end_datetime_idx'),
+            BrinIndex(fields=['created'], name='item_created_idx'),
+            models.Index(fields=['updated'], name='item_updated_idx'),
```

Comment:
Are those really required ? For which use case ? And why only have the index on the Items and not on the collections and Assets ?

Reply:
This is due to `queriable_date_fields = ['datetime', 'created', 'updated']` in `views.py`